### PR TITLE
engine(metrics): remove jobtype prefix in metrics

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### What problem does this PR solve?

Issue Number: ref #6985

### What is changed and how it works?
1. remove jobtype prefix in metrics

#### Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by lance6716 — COMMENTED** [review]
what's the effect before and after this PR?

File: engine/framework/worker.go [hunk 1]
```diff
@@ -236,8 +236,9 @@ func NewBaseWorker(
 		errCenter:            errctx.NewErrCenter(),
 		clock:                clock.New(),
 		businessMetaKVClient: cli,
-		metricFactory:        promutil.NewFactory4Worker(ctx.ProjectInfo, MustConvertWorkerType2JobType(tp), masterID, workerID),
-		logger:               frameLog.WithWorkerID(frameLog.WithMasterID(logger, masterID), workerID),
+		metricFactory: promutil.NewFactory4Worker(ctx.ProjectInfo,
```
Comment (amyangfei) [inline engine/framework/worker.go:239]: Is it better to change the function signature to 
```go
func NewFactory4Worker(info tenant.ProjectInfo, prefix string, jobID engineModel.JobID, workerID frameModel.WorkerID,
```
Then we don't need to add the job type `JobTypeNameNone`

**Comment by CharlesCheung96** [comment]
> what's the effect before and after this PR?

@lance6716  Removing the type prefix makes it easier to reuse the existing monitoring panel of the dm.

**Review by amyangfei — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Pull request merged**